runtime/foucoco/src/lib.rs

```diff
@@ -706,11 +749,13 @@ impl orml_tokens::Config for Runtime {
 
 parameter_types! {
 	pub const NativeCurrencyId: CurrencyId = CurrencyId::Native;
+	pub const WrappedCurrencyId: CurrencyId = WRAPPED_CURRENCY_ID;
```

Comment:
Do we need this? I don't think we need this anymore. I guess it's actually an orphan statement in the spacewalk testchain I might have forgotten to remove. 🤫

Reply:
https://github.com/pendulum-chain/pendulum/pull/91/files#diff-63fad4ac7944e1d379fd236f19986bfb4f8c39eb199a14535775de9a81d72a08R1332

https://github.com/pendulum-chain/pendulum/pull/91/files#diff-f2256017c4e606a30f53708578da46675512cc1300be6542ed5f77e58af0d6dcR581

I think from these

Reply:
Ah okay, interesting. Well, that makes sense actually. But then let's maybe rename it since it merely acts as a default value.
```
--- pub const WrappedCurrencyId: CurrencyId = WRAPPED_CURRENCY_ID;
+++ pub const DefaultWrappedCurrencyId: CurrencyId = WRAPPED_CURRENCY_ID;
```

Reply:
done